NitroxClient/GameLogic/Items.cs

```diff
@@ -41,7 +42,11 @@ public void PickedUp(GameObject gameObject, TechType techType)
             // We want to remove any remote tracking immediately on pickup as it can cause weird behavior like holding a ghost item still in the world.
             RemoveAnyRemoteControl(gameObject);
 
-            NitroxId id = NitroxEntity.GetId(gameObject);
+            if (!gameObject.TryGetNitroxId(out NitroxId id))
```

Comment:
Let's verify that the id exists already for picking up items from a crafter and that we weren't using GetId for that.

Reply:
Sry I don't get what you are saying here. Could maybe explain?

Reply:
I was saying that this may have been assigning an id when a user was initially picking up an item from a crafter.  Can you test picking up an item from a crafter and ensure it works?